plugins/org.jboss.tools.openshift.ui/src/org/jboss/tools/openshift/internal/ui/job/DeployImageJob.java

```diff
@@ -124,18 +124,26 @@ protected IStatus doRun(IProgressMonitor monitor) {
 		return Status.OK_STATUS;
 	}
 
-	private boolean isUpdate(Connection connection, String project, String name) {
+	private boolean isUpdateThenDo(Connection connection, String project, String name) {
```

Comment:
isUpdate then do what? I agree that isUpdate() isnt a good choice for a method name, but I still dont get (by reading the method signature) what this method does.
Is this something like "updateTriggerIfIsUpdate" or similar?